src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -493,5 +564,16 @@ private HttpResponseMessage CreateCheckStatusResponseMessage(HttpRequestMessage
             response.Headers.RetryAfter = new RetryConditionHeaderValue(TimeSpan.FromSeconds(10));
             return response;
         }
+
+        private TimeSpan? GetTimeSpan(HttpRequestMessage request, string queryParameterName)

```

Comment:
Methods which do not reference instance state should be marked as `static`. You can find some good discussion about this best practice here: https://stackoverflow.com/questions/169378/method-can-be-made-static-but-should-it